trunner/target/host.py

```diff
@@ -63,6 +60,7 @@ def fail(result: TestResult):
             builder.add(fail)
             return builder.get_harness()
 
+        test.shell.cmd[0] = f"{self.root_dir()}{test.shell.cmd[0]}"
```

Comment:
Some comment may be helpful there
```suggestion
        # on host we execute binaries directly from the root directory located in phoenix-rtos-project/_fs
        test.shell.cmd[0] = f"{self.root_dir()}{test.shell.cmd[0]}"
```